### ISSUE TITLE
Icon color is different when installing to other than `default-release`

### ISSUE THREAD

**Issue by RShirohara** [vinceliuice/WhiteSur-firefox-theme#44]
## Summary

I usually create and work with a separate profile other than `default-release` and apply this theme through manual installation.  
However, manually installed profiles do not display the correct color of icons, resulting in black icons on a dark background.

## Expected Behavior

A white icons appear on a dark background.

![image](https://github.com/user-attachments/assets/f3de2313-b305-4222-b5de-fd0d66280aa7)

## Actual Behavior

A black icon appears on a dark background.

![image](https://github.com/user-attachments/assets/96694980-ba66-446d-9f3f-97419d6982af)

## Steps to Reproduce

1. Install and launch Firefox.
2. Install this theme.
3. Create a new profile in Firefox and activate the setting `toolkit.legacyUserProfileCustomizations.stylesheets`.
4. Copy the `chrome` folder from the `default-release` to the new profile.
5. Launch Firefox with a new profile.

## Environment

- OS: ArchLinux
- Browser: Firefox 132.0.1
  - Reproduced in both `pacman` and `flatpak` versions.

## Additional Info

Since I am not a native English speaker, this issue was created using DeepL Translate and Google Translate.

**Comment by vinceliuice** [comment]
Reinstall it again

**Comment by RShirohara** [comment]
OK.

I installed this theme again, this time copying the CSS generated in `firefox-themes` to the new profile.

But the result seems to be the same.

![image](https://github.com/user-attachments/assets/c363e9a6-2700-49f5-8549-93d908e14dd4)

**Comment by vinceliuice** [comment]
Did it not installed in your profile folder ? why need copy it ? this install.sh will install this theme for all profiles

**Comment by RShirohara** [comment]
> Did it not installed in your profile folder ? why need copy it ? this install.sh will install this theme for all profiles

As far as I have tried with the `flatpak` version, it is only installed on the `default-release` profile and not on the other profiles added by the profile manager.

> Also you need copy user.js file

I did not notice this!
I copied the `user.js` and now the icons appear correctly.
Thank you.

![image](https://github.com/user-attachments/assets/ecb24b63-0778-4011-9f3c-5cdd9c6a1f0a)
